tests/base/test_featurizers.py

```diff
@@ -141,3 +144,33 @@ def test_count_vector_featurizer(sentence, expected):
     ftr.process(message)
 
     assert np.all(message.get("text_features")[0] == expected)
+
+
+@pytest.mark.parametrize("tokens, expected", [
+    (["hello", "hello", "hello", "hello", "hello"], [5]),
+    (["你好", "你好", "你好", "你好", "你好"], [5]),  # test for unicode chars
+    (["hello", "goodbye", "hello"], [1, 2]),
+
+    # Note: order has changed in Chinese version of "hello" & "goodbye"
+    (["你好", "再见", "你好"], [2, 1]),  # test for unicode chars
+    (["a", "b", "c", "d", "e", "f"], [1, 1, 1, 1, 1, 1]),
+    (["a", "1", "2"], [2, 1])
+])
+def test_count_vector_featurizer_using_tokens(tokens, expected):
+    from rasa_nlu.featurizers.count_vectors_featurizer import \
+        CountVectorsFeaturizer
+
+    ftr = CountVectorsFeaturizer({"token_pattern": r'(?u)\b\w+\b'})
+
+    message = Message("")
```

Comment:
Could you please add a comment why `message` is empty string here. I get it now, but I'm not sure it will be that easy to grasp in one month.

Reply:
Sure, I will fix it very soon.